.github/workflows/lint_python.yml

```diff
@@ -15,8 +15,8 @@ jobs:
       - run: bandit --recursive --skip B105,B110,B311,B605,B607 .
       - run: black --check .
       - run: codespell  # --ignore-words-list="" --skip=""
-      - run: flake8 --ignore=E12,E20,E226,E231,E265,E271,E40,E731,F401,W29,W605
-                    --max-complexity=10 --max-line-length=255 --show-source --statistics .
+      - run: flake8 --ignore=E203 --max-complexity=10 --max-line-length=255
```

Comment:
We now use black which has a 88 max character line length. We could set the `max-line-length` to 88 too.

Unless it is applied to comments and doc etc 🤔

Reply:
Comments.  If you lower the number, you will see which lines.